.changeset/hot-coins-pretend.md

```diff
@@ -0,0 +1,9 @@
+---
+"@project44-manifest/react": patch
```

Comment:
this should be a minor release since we are adding backwards compatible new functionality: https://semver.org/

Reply:
Ok perfect